homeassistant/components/yale_smart_alarm/lock.py

```diff
@@ -45,7 +45,7 @@ def __init__(
     ) -> None:
         """Initialize the Yale Lock Device."""
         super().__init__(coordinator, data)
-        self._attr_code_format = f"^\\d{code_format}$"
+        self._attr_code_format = f"^\\d{{{code_format}}}$"
```

Comment:
Why is the extra backslash there btw?

Shouldn't it be:

`^\d{{{code_format}}}$`?

Reply:
You're right. It's a typo from before which now came back as I copied the previous text. 
Will remove it as soon as I get behind the laptop again.